tee/prototype/epochproc.go

```diff
@@ -0,0 +1,397 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package prototype
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	err "perun.network/go-perun/pkg/errors"
```

Comment:
I wouldn't use `err`, it's the usual error variable name. Could lead to unintended shadowing. Bette like `perrors`.

Reply:
good point.